filament/backend/src/opengl/GLUtils.cpp

```diff
@@ -28,84 +28,68 @@ using namespace utils;
 namespace GLUtils {
 
 UTILS_NOINLINE
-const char* getGLError(GLenum error) noexcept {
-    const char* string = "unknown";
+const char* getGLError(const GLenum error) noexcept {
```

Comment:
I prefer this method to have a single `return` statement so it's possible to set a breakpoint on it.

Reply:
That's fine, but then there's no standard across similar functions.
Other functions are using early return in switch statements.